fugue_test/builtin_suite.py

```diff
@@ -465,6 +465,33 @@ def test_distinct(self):
                     )
                 )
 
+        def test_dropna(self):
```

Comment:
I really like these tests. They are very readable, like an instruction of the function. Great job!